modules/pks/variables.tf

```diff
@@ -14,3 +14,7 @@ locals {
   pks_cidr          = "${cidrsubnet(var.resource_group_cidr, 6, 3)}"
   pks_services_cidr = "${cidrsubnet(var.resource_group_cidr, 6, 4)}"
 }
+
+variable "bosh_deployed_vms_security_group_id" {
+  default = ""
```

Comment:
Since this is an internal variable (i.e. in a module and not exposed), can we remove the default? Looks like it will always be set if this module is used

Reply:
Yeah let me remove it thanks